packages/waku/src/lib/handlers/handler-dev.ts

```diff
@@ -228,9 +228,15 @@ export function createHandler<
         item.url !== viteUrl &&
         !item.url.includes('?html-proxy')
       ) {
+        const { code } = (await vite.transformRequest(item.url))!;
         res.setHeader('Content-Type', 'application/javascript');
         res.setStatus(200);
-        endStream(res.stream, `export * from "${item.url}";`);
```

Comment:
It might not follow ESM, but does this cause an error?
```ts
endStream(res.stream, `export * from "${item.url}"; export { default } from "${item.url}";`);
```

Reply:
Yep, I haven't tried it, but the reason I guess is that the default export does not exist in many cases.

Reply:
And exporting what is not exported would cause an error I assume.

Reply:
yeah, that's what I've concerned. If you tried it, it's good.